### PR TITLE
change get_merged_prs to be the graphql version

### PR DESCRIPTION
**Pull request by eteq**
This PR just removes the "old" get_merged_prs" in favor of the graphql version @astrofrog  implemented.

@astrofrog, can you have a look and confirm there's not some compelling reason I missed not to do this?

### CODE REVIEW

File: pr_consistency/1.get_merged_prs.py [hunk 1]
```diff
@@ -1,116 +1,105 @@
...
-        labels = [label.name for label in issue.labels]
-
-        pull_requests[str(pr.number)] = {'milestone': milestone,
-                                         'title': pr.title,
-                                         'labels': labels,
-                                         'merged': pr.merged_at.isoformat(),
-                                         'updated': pr.updated_at.isoformat(),
-                                         'created': pr.created_at.isoformat(),
-                                         'merge_commit': pr.merge_commit_sha}
+            if pr['number'] == 1041:
```
Comment (bsipocz) [inline pr_consistency/1.get_merged_prs.py:87]: why is 1041 special?

File: pr_consistency/1.get_merged_prs.py [hunk 2]
```diff
@@ -1,116 +1,105 @@
...
 
 print("The repository this script currently works with is '{}'.\n"
       .format(REPOSITORY))
 
+json_filename = 'merged_pull_requests_{}.json'.format(NAME)
 
-# Get handle to repository
-g = Github(*get_credentials())
-repo = g.get_repo(REPOSITORY)
+TOKEN = input('Enter your GitHub personal token: ')
```
Comment (bsipocz) [inline pr_consistency/1.get_merged_prs.py:55]: does this require the token to be entered each time, or will reuse it like the previous version did?

Comment (eteq) [reply pr_consistency/1.get_merged_prs.py:55]: it requires it every time... which now that you mention it is perhaps a reason not to make this change just yet!

File: pr_consistency/1.get_merged_prs.py (hunk 1 above)
Comment (eteq) [reply pr_consistency/1.get_merged_prs.py:87]: not clear to me - maybe @astrofrog has insight? (as the original author here)

**Review by astrofrog — APPROVED** [review]
Yep looks good!

File: pr_consistency/1.get_merged_prs.py (hunk 1 above)
Comment (astrofrog) [reply pr_consistency/1.get_merged_prs.py:87]: Leftover debug statement!

File: pr_consistency/1.get_merged_prs.py (hunk 2 above)
Comment (astrofrog) [reply pr_consistency/1.get_merged_prs.py:55]: Unlike the previous method this requires a token, no way to do it with username/password. We could add ways of specifying it permanently though, env var, hidden file, etc.

Comment (bsipocz) [reply pr_consistency/1.get_merged_prs.py:55]: The previous version also used a token, at least as I recall I used it with a token to work around 2fa (can check on it tomorrow evening)

File: pr_consistency/1.get_merged_prs.py (hunk 1 above)
Comment (eteq) [reply pr_consistency/1.get_merged_prs.py:89]: Will remove!

**Comment by eteq** [comment]
I updated this now to use the `get_credential` mechanism - as @bsipocz said it used tokens, but was unclear that this was the intent.  I tried to clarify that explicitly in the commits I just pushed up require explicitly saying when you want a token in `get_credential`.

**Comment by astrofrog** [comment]
I don't think the GraphQL works without a token but I could be wrong.

**Comment by bsipocz** [comment]
I do have a token (you can't use a passwd with 2FA, which is non-optional for me), but it's stored and used automatically with a current solution, which would be nice here, too.

**Comment by astrofrog** [comment]
@bsipocz - ah right of course I see, yes I agree that would be nice. Do your latest changes make that work @eteq?

**Comment by eteq** [comment]
@astrofrog @bsipocz - yes  0d96e1be9bd60b6ffa132087c5ed9a4268470c24 changes it so that the "password" machinery from before is treated as a token, and lets you use the `netrc` to store it.

**Pull request merged**